modules/era/sql/era_droplist.sql

```diff
@@ -19989,43 +20013,43 @@ INSERT INTO `mob_droplist` VALUES (2508,0,0,1000,18948,@UNCOMMON); -- Enforcer (
-INSERT INTO `mob_droplist` VALUES (2511,0,0,1000,658,@RARE);      -- Damascus Ingot (Rare, 5%)
-INSERT INTO `mob_droplist` VALUES (2511,0,0,1000,702,@RARE);      -- Ebony Log (Rare, 5%)
-INSERT INTO `mob_droplist` VALUES (2511,0,0,1000,738,@RARE);      -- Chunk Of Platinum Ore (Rare, 5%)
-INSERT INTO `mob_droplist` VALUES (2511,0,0,1000,823,@RARE);      -- Spool Of Gold Thread (Rare, 5%)
-INSERT INTO `mob_droplist` VALUES (2511,0,0,1000,830,@RARE);      -- Square Of Rainbow Cloth (Rare, 5%)
-INSERT INTO `mob_droplist` VALUES (2511,0,0,1000,836,@RARE);      -- Square Of Damascene Cloth (Rare, 5%)
-INSERT INTO `mob_droplist` VALUES (2511,0,0,1000,866,@RARE);      -- Handful Of Wyvern Scales (Rare, 5%)
-INSERT INTO `mob_droplist` VALUES (2511,0,0,1000,895,@RARE);      -- Ram Horn (Rare, 5%)
-INSERT INTO `mob_droplist` VALUES (2511,0,0,1000,902,@RARE);      -- Demon Horn (Rare, 5%)
-INSERT INTO `mob_droplist` VALUES (2511,0,0,1000,1110,@RARE);     -- Vial Of Black Beetle Blood (Rare, 5%)
-INSERT INTO `mob_droplist` VALUES (2511,0,0,1000,4175,@RARE);     -- Vile Elixir +1 (Rare, 5%)
-INSERT INTO `mob_droplist` VALUES (2511,2,0,1000,751,0);          -- Platinum Beastcoin (Steal)
-INSERT INTO `mob_droplist` VALUES (2511,0,1,1000,17619,750);      -- Daylight Dagger (Group 1 - 75.0%)
-INSERT INTO `mob_droplist` VALUES (2511,0,1,1000,17528,250);      -- Astral Signa (Group 1 - 25.0%)
+INSERT INTO `mob_droplist` VALUES (2511,0,0,1000,751,@ALWAYS);   -- Platinum Beastcoin (Always, 100%)
+INSERT INTO `mob_droplist` VALUES (2511,0,0,1000,17619,@ALWAYS); -- Daylight Dagger (Always, 100%)
+INSERT INTO `mob_droplist` VALUES (2511,0,0,1000,1440,@ALWAYS);  -- Summoners Testimony (Always, 100%)
+INSERT INTO `mob_droplist` VALUES (2511,0,0,1000,844,@VCOMMON);  -- Phoenix Feather (36.0%)
+INSERT INTO `mob_droplist` VALUES (2511,0,0,1000,1132,@VCOMMON); -- Square Of Raxa (31.0%)
+INSERT INTO `mob_droplist` VALUES (2511,0,0,1000,942,@COMMON);   -- Philosophers Stone (23.0%)
+INSERT INTO `mob_droplist` VALUES (2511,0,0,1000,1132,@COMMON);  -- Square Of Raxa (Common, 15%)
+INSERT INTO `mob_droplist` VALUES (2511,0,0,1000,737,@UNCOMMON); -- Chunk Of Gold Ore (13.0%)
+INSERT INTO `mob_droplist` VALUES (2511,0,0,1000,700,@RARE);     -- Mahogany Log (Uncommon, 10%)
+INSERT INTO `mob_droplist` VALUES (2511,0,0,1000,703,@UNCOMMON); -- Petrified Log (Uncommon, 10%)
+INSERT INTO `mob_droplist` VALUES (2511,0,0,1000,887,@RARE);     -- Coral Fragment (Rare, 5%)
+INSERT INTO `mob_droplist` VALUES (2511,0,0,1000,837,@RARE);     -- Spool Of Malboro Fiber (8.0%)
+INSERT INTO `mob_droplist` VALUES (2511,0,0,1000,902,@RARE);     -- Demon Horn (8.0%)
+INSERT INTO `mob_droplist` VALUES (2511,0,0,1000,1110,@RARE);    -- Vial Of Black Beetle Blood (8.0%)
+INSERT INTO `mob_droplist` VALUES (2511,0,0,1000,4173,@RARE);    -- Hi-Reraiser (8.0%)
+INSERT INTO `mob_droplist` VALUES (2511,0,0,1000,17528,@RARE);   -- Astral Signa (7.0%)
```

Comment:
Daylight Dagger and Astral Signa should be a group drop rate as only one of them can drop a a time. As for the drop rate probably 95%/5?